content/test/integration/controller/deployment-pagination.spec.ts

```diff
@@ -0,0 +1,323 @@
+import { Bean } from '@katalyst/content/Environment'
+import { DeploymentOptions, PointerChangesFilters } from '@katalyst/content/service/deployments/DeploymentManager'
+import { isSuccessfulDeployment } from '@katalyst/content/service/Service'
+import { MockedSynchronizationManager } from '@katalyst/test-helpers/service/synchronization/MockedSynchronizationManager'
+import assert from 'assert'
+import { EntityType, Fetcher, SortingField, SortingOrder, Timestamp } from 'dcl-catalyst-commons'
+import {
+  toQueryParamsForGetAllDeployments,
+  toQueryParamsForPointerChanges
+} from 'decentraland-katalyst-commons/QueryParameters'
+import { loadStandaloneTestEnvironment } from '../E2ETestEnvironment'
+import { buildDeployData, EntityCombo } from '../E2ETestUtils'
+import { TestServer } from '../TestServer'
+
+describe('Integration - Deployment Pagination', () => {
+  let E1: EntityCombo, E2: EntityCombo, E3: EntityCombo
+
+  const testEnv = loadStandaloneTestEnvironment()
+  let server: TestServer
+  const fetcher = new Fetcher()
+
+  beforeEach(async () => {
+    server = await testEnv
+      .configServer()
+      .withBean(Bean.SYNCHRONIZATION_MANAGER, new MockedSynchronizationManager())
+      .andBuild()
+    await server.start()
+  })
+
+  beforeAll(async () => {
+    const P1 = 'X1,Y1'
+    const P2 = 'X2,Y2'
+    const P3 = 'X3,Y3'
+    const type = EntityType.PROFILE
+
+    const timestamp = Date.now()
+    const a = await buildDeployData([P1], { type, timestamp, metadata: 'metadata1' })
+    const b = await buildDeployData([P2], { type, timestamp, metadata: 'metadata2' })
+    if (a.entity.id.toLowerCase() < b.entity.id.toLowerCase()) {
+      E1 = a
+      E2 = b
+    } else {
+      E1 = b
+      E2 = a
+    }
+    const laterTimestamp = Date.now()
+    E3 = await buildDeployData([P3], { type, timestamp: laterTimestamp, metadata: 'metadata3' })
+  })
+
+  it('When there is no next page then the link is undefined', async () => {
+    // Deploy E1, E2 and E3 in that order
+    await deploy(E1, E2, E3)
+
+    const actualDeployments = await fetchDeployments({
+      limit: 3
+    })
+
+    expect(actualDeployments.deployments.length).toBe(3)
+    const nextLink = actualDeployments.pagination.next
+    expect(nextLink).toBeUndefined()
+  })
+
+  it('When local timestamp filter is set, then only toLocalTimestamp is modified in next ', async () => {
+    // Deploy E1, E2 and E3 in that order
+    const [E1Timestamp, E2Timestamp, E3Timestamp] = await deploy(E1, E2, E3)
+
+    const actualDeployments = await fetchDeployments({
+      limit: 2,
+      filters: { fromLocalTimestamp: E1Timestamp, toLocalTimestamp: E3Timestamp }
```

Comment:
I would recommend using `to` and `from` with the `sortBy`. That will make deprecation easier

(Also in other tests)

Reply:
I wanted to be sure that it still works, I'll add a separate test for that then.